scripts/build/download-tool.sh

```diff
@@ -78,7 +78,7 @@ if [ -f "$ARGV_OUTPUT" ]; then
   fi
 fi
 
-wget --no-check-certificate "$ARGV_URL" -O "$TEMP_OUTPUT"
+wget --no-verbose --no-check-certificate "$ARGV_URL" -O "$TEMP_OUTPUT" 2>&1
```

Comment:
Not necessarily related, but is there a specific reason we'd want `--no-check-certificate` here (and below)? Considering we're doing production builds with this, we should error on broken / invalid certs, imo.